images/nginx/rootfs/build.sh

```diff
@@ -376,6 +376,12 @@ Include /etc/nginx/owasp-modsecurity-crs/rules/RESPONSE-999-EXCLUSION-RULES-AFTE
 # build nginx
 cd "$BUILD_PATH/openresty-$OPENRESTY_VERSION"
 
+echo "Patching NGINX for CVE-2018-16843, CVE-2018-16844, CVE-2019-9511, CVE-2019-9513, and CVE-2019-9516"
+# Upstream change https://github.com/openresty/openresty/pull/515
+# TODO: remove after openresty release
+cat /patches/patch.2019.h2.txt | patch -d bundle/nginx-1.15.8/ -p0
```

Comment:
Should this not be a relative path `patches/...` instead of `/patches/...`

Reply:
oh well I guess this will work because we copy everything in `rootfs` to `/` during image build

Reply:
correct